polymath/extensions/indexer.py

```
@@ -4,588 +4,592 @@
+
+        return obj
+
+    def __setitem__(self, indx, arg):
+
+        self.require_writeable()
+
+        # Handle indexing of a shapeless object, and indices consistent with
+        # shapeless indexing
+        try:
+            (masked, size_zero,
+             shape_before, shape_after) = self._prep_scalar_index(indx)
```

Ruff (0.14.7)

Unpacked variable `shape_before` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)